src/components/plots/barPlot.ts

```diff
@@ -315,6 +351,9 @@ export module Plots {
     }
 
     private _updateValueScale() {
+      if (!this._projectorsReady()) {
+        return;
+      }
```

Comment:
Why is this here?  Wouldn't a better catch be to move this one line below and break out if valueScale is null?